specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json

```diff
@@ -1284,49 +1082,67 @@
           "items": {
             "$ref": "#/definitions/error"
           },
-          "description": "An array of details about specific errors that led to this reported error."
+          "description": "An array of details about specific errors that led to this reported error.",
+          "readOnly": true
         },
         "target": {
-          "description": "The target of the error",
-          "type": "string"
+          "description": "The target of the particular error (e.g., the name of the property in error).",
+          "type": "string",
+          "readOnly": true
         },
-        "innererror": {
+        "innerError": {
```

Comment:
This needs to be x-nullable, as does "details".

Reply:
we will fix this on the service side by not emitting null values there